### PR TITLE
docs: `ecosystem/integration` update

### PR DESCRIPTION
**Pull request by leo-gan**
The example notebooks with `CallbackHandlers` examples were in the  `ecosystem/integration` folder. 
They should be in the `modules/callbacks/integrations` folder.

- Moved notebooks. See ^.
- Created correspondent  `ecosystem/integration` pages in place of examples. Now no Jupyter Notebooks are here, but only .md and .mdx .
- Added descriptions to the Integration Cards.

@baskaryan

### CODE REVIEW

**Comment by baskaryan** [comment]
my main concern is most of these integrations pages/notebooks were written by the owners of the integrations, and generally lean towards letting them own the content around their product

**Comment by leo-gan** [comment]
> my main concern is most of these integrations pages/notebooks were written by the owners of the integrations, and generally lean towards letting them own the content around their product

Exactly. That's why I didn't change the notebooks in the code and explanation parts. I just added descriptions when they were missed and made the format consistent with our template.
 
 But we do not hold examples in the `ecosystem/integration` section. Pages in this section (let's call them the `Integration Cards` as HuggingFace :) ) represent only top-level information: description, installation, classes based on this integration, and links to the class examples. 
 
To summarize: 
 `Examples` are created by the integration creators/owners with very little control on the content and a little more control on the format. `Examples` are saved into the correspondent Module folders.
 
 `Integration cards` are under our control and have strict content and formatting rules. That allows us to automate our docs with these Cards.

**Comment by leo-gan** [comment]
@baskaryan Please, give me feedback.

**Comment by leo-gan** [comment]
Closing it because nobody reviews it.

**Comment by leo-gan** [comment]
Closing it because nobody reviews it.
